LICENSE

```diff
@@ -0,0 +1,24 @@
+MIT License
+
+Copyright (c) 2018   
```

Comment:
We should review this - as we use Crown Copyright which is special - I will add a ticket to the backlog to review this.